test_setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
can you give more context for this file? Is it exclusively for testing, or will we have to update it ever?

Additionally, does it run in an automated manner?

Reply:
Stray file, removed.